api/v1alpha1/groupversion_info.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 const (
-	Group          = "metaclusterinstall.openshift.io"
+	Group          = "cluster.open-cluster-management.io"
```

Comment:
hmm, did we decide on _cluster_.open-cluster-management.io? I thought we just said _something_ in `open-cluster-management.io`.

If we're leaving open the possibility that this group could include non-cluster things maybe we should stick with `siteconfig.open-cluster-management.io` just to match the operator?

cc @jnpacker what do you remember about this one?

Reply:
This is the APIGroup @jnpacker specified in the meeting notes. Regarding non-cluster objects, I believe we agreed that they would be created as separate CRDs. These objects may also follow suit and add a prefix to `open-cluster-management.io` with the object-type. Having said that, I will let @jnpacker weigh-in on this one.

Reply:
My understanding is that the API group is shared across all the resources defined in this package. It needs to not be tied to a specific kind in the API.

Reply:
As discussed, we will go with `siteconfig.open-cluster-management.io` as the APIGroup. Will update the PR accordingly.